client/a8c-for-agencies/components/layout/top.tsx

```diff
@@ -19,6 +20,7 @@ export default function LayoutTop( { children, withNavigation }: Props ) {
 				'has-navigation': withNavigation || !! navigation,
 			} ) }
 		>
+			<PendingPaymentNotification />
```

Comment:
As we want to show it on all places, I think it's ok to have it for all layouts.
My only concern is possible call to `invoices` endpoint everytime, but if it would be a problem we could probably move it to the Store.